apps/microtvm/cmsisnn/src/demo_bare_metal.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdio.h>
+#include <tvm_runtime.h>
+#include <tvmgen_detection.h>
+
+#include "uart.h"
+
+// Header files generated by convert_image.py
+#include "inputs.h"
+#include "outputs.h"
+
+int abs(int v) { return v * ((v > 0) - (v < 0)); }
```

Comment:
Just for curiosity, is there a particular reason why abs is implemented this way?
That is, for example, instead of `v >= 0 ? v : -v`.

Reply:
Thanks for drawing my attention to this @NicolaLancellotti.
This code was copied from the microNPU demo and to be honest I'm not sure why it was done that way.
I did notice though that the CMSIS-NN only demo doesn't make use of the abs() function, so I've removed it now.